src/components/Notifications/models.tsx

```diff
@@ -29,3 +29,14 @@ export interface PNconfigData {
   pf_endpoint: string;
   pf_p256dh: string;
 }
+
+export interface NoticeData {
+  id: string;
+  caused_objects: cause_object;
+  caused_by: UserBareMinimum;
+  event: string;
+  event_type: string;
+  created_date: string;
+  read_at: string;
+  message: string;
+}
```

Comment:
why do we need a new type def? can't we use the same as one that'd type inferred by the route def. when using useQuery?